.github/workflows/commit_artifacts.yml

```diff
@@ -2,7 +2,7 @@ name: Commit Artifacts for Facebook WWW
 
 on:
   push:
-    branches: [main]
+    branches: [main, lt/difftrain-title]
```

Comment:
revert?

Reply:
Yeah this was just a testing commit so I could verify it worked, reverted!